feature-libs/product-configurator/common/components/configure-product/configure-product.component.spec.ts

```diff
@@ -57,7 +59,11 @@ function setupWithCurrentProductService(
 ) {
   if (useCurrentProductServiceOnly && currenProductServiceReturnsNull) {
     TestBed.configureTestingModule({
-      imports: [I18nTestingModule, RouterTestingModule],
+      imports: [
+        I18nTestingModule,
+        RouterTestingModule,
```

Comment:
`RouterTestingModule,` and `StoreModule.forRoot({})` are not required. What we need is:
```
      {
          provide: RoutingService,
          useClass: MockRoutingService,
        },
```
and 
```
class MockRoutingService implements Partial<RoutingService> {
  go() {}
}
```

Reply:
picked

Reply:
done